auction-server/config.sample.yaml

```diff
@@ -18,3 +18,4 @@ chains:
     accepted_token_programs:
       - TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA
       - TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb
+    associated_token_program_id: ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL
```

Comment:
mehhh, why not harcode this: I don't think this will ever change

Reply:
for other SVM chains in the future potentially

Reply:
not a big deal but i think this is good practice